packages/perseus/src/widgets/phet-simulation/phet-simulation.tsx

```diff
@@ -185,18 +185,25 @@ export class PhetSimulation extends React.Component<Props, State> {
                         />
                     </View>
                 )}
-                <iframe
-                    ref={this.iframeRef}
-                    title={this.props.description}
-                    sandbox={sandboxProperties}
-                    style={{
-                        height: 225,
-                        width: "100%",
-                        borderWidth: 0,
-                    }}
-                    src={this.state.url?.toString()}
-                    allow="fullscreen"
-                />
+                <View style={styles.iframeContainer}>
+                    <iframe
+                        ref={this.iframeRef}
+                        title={this.props.description}
+                        sandbox={sandboxProperties}
+                        style={{
+                            borderWidth: 0,
+                            position: "absolute",
+                            top: 0,
+                            left: 0,
+                            bottom: 0,
+                            right: 0,
+                            width: "100%",
+                            height: "100%",
+                        }}
```

Comment:
We should probably move these styles to Aphrodite below! I'll make a note here and we can do it in a cleanup ticket.

Reply:
Done! I had been having some problems getting the Aphrodite styling to work on a non-Wonder Blocks element, but I just got it to work, so I'll commit that update now:)

Reply:
@aemandine You can make the `iframe` into a component that supports aphrodite styling by using the `addStyle`. 

See: https://khan.github.io/wonder-blocks/?path=/docs/packages-core-addstyle--docs